src/mcp/README.md

```diff
@@ -34,26 +34,43 @@ In `.cursor/mcp.json` in your workspace directory, add the Firebase MCP server:
-| `auth_set_claims`                | `auth`        | Sets custom claims on a specific user's account. Use to create trusted values associated with a user e.g. marking them as an admin. Claims are limited in size and should be succinct in name and value.       |
-| `auth_set_sms_region_policy`     | `auth`        | Sets an SMS Region Policy for Firebase Auth to restrict the regions which can receive text messages based on an ALLOW or DENY list of country codes. This policy will override any existing policies when set. |
-| `dataconnect_list_services`      | `dataconnect` | List the Firebase Data Connect services available in the current project.                                                                                                                                      |
-| `dataconnect_generate_schema`    | `dataconnect` | Generates a Firebase Data Connect Schema based on the users description of an app.                                                                                                                             |
-| `dataconnect_generate_operation` | `dataconnect` | Generates a single Firebase Data Connect query or mutation based on the currently deployed schema and the provided prompt.                                                                                     |
-| `dataconnect_get_schema`         | `dataconnect` | List the Firebase Data Connect Schema in the project, which includes Cloud SQL data sources and the GraphQL Schema describing what tables are available.                                                       |
-| `dataconnect_get_connector`      | `dataconnect` | Get the Firebase Data Connect Connectors in the project, which includes the pre-defined GraphQL queries accessible to client SDKs.                                                                             |
-| `storage_get_rules`              | `storage`     | Retrieves the Firebase Cloud Storage Rules for the default bucket.                                                                                                                                             |
-| `send_message_to_fcm_token`      | `messaging`   | Sends a push message to an FCM Token.                                                                                                                                                                          |
-| `send_message_to_fcm_topic`      | `messaging`   | Sends a push message to an FCM Topic.                                                                                                                                                                          |
+Generate the  below table using the following command:
+
+```bash
+firebase experimental:mcp --generate-tool-docs
+```
+
```

Comment:
```suggestion
```

We don't need to document how to do this - it should be for our own internal use.

Reply:
How about `CONTRIBUTING.md` in the mcp directory since that's where contribution guides go for GH generally.